applications/isaac_holoscan_bridge/README.md

```diff
@@ -44,7 +45,7 @@ flowchart LR
 ## Run Instructions
 
 ```bash
-./dev_container build_and_run --container_launch_args "--as_root" --container_args "-e ACCEPT_EULA=Y -e PRIVACY_CONSENT=Y \
+./holohub run --as-root --docker-opts "-e ACCEPT_EULA=Y -e PRIVACY_CONSENT=Y \
```

Comment:
Might need `=`. I think we also determined `-u root` will replace `--as-root`, cc @wyli  to confirm.
```suggestion
./holohub run --docker-opts="-u root -e ACCEPT_EULA=Y -e PRIVACY_CONSENT=Y \
```